site/en/blog/no-spooky-cookies/index.md

```diff
@@ -0,0 +1,204 @@
+---
+title: No spooky cookies
+subhead: >
+  Cookies are best fresh, so what are the latest recipes to ensure you can
+  still enjoy spooky season without any stale cookies?
+description: >
+  Cookies are best fresh, so what are the latest recipes to ensure you can
+  still enjoy spooky season without any stale cookies?
+layout: 'layouts/blog-post.njk'
+date: 2022-10-24
+authors:
+  - rowan_m
+hero: 'image/VWw0b3pM7jdugTkwI6Y81n6f5Yc2/m2MdomFImr4xrDeBgRht.png'
+alt: >
+  A random assortment of spider webs, hourglasses and cookies - as time is
+  running out for third-party cookies on the web. Ho ho ho.
+tags:
+  - privacy
+---
+
+Cookies are always best when they are fresh, so what are the latest recipes to
+ensure you can still enjoy spooky season without any spooky, stale cookies?
```

Comment:
```suggestion
ensure you can still enjoy spooky season without any spooky stale cookies?
```

Reply:
Actually, will just remove the double spooky here. I don't like it on re-reading.